flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/consumer/SingleInputGateTest.java

```diff
@@ -60,10 +61,7 @@
 import java.io.IOException;
 import java.nio.ByteBuffer;
 import java.nio.ByteOrder;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Optional;
+import java.util.*;
```

Comment:
Wildcard imports are [discouraged](https://flink.apache.org/contributing/code-style-and-quality-formatting.html#imports) by flink style guide.

Reply:
Sure, my local IDE did it automatically.